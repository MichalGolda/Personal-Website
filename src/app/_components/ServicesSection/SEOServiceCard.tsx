import ServiceCard from "./ServiceCard";
import ServiceIcon from "./ServiceIcon";

const SEOServiceIcon = () => <ServiceIcon src="/seo.webp" alt="seo" />;

export const SEOServiceCard = () => (
  <ServiceCard
    title="Optymalizacja treści (SEO) oraz szybkości ładowania"
    content={
      <>
        Badanie słów kluczowych i ich strategiczne umieszczanie w treściach, aby
        poprawić widoczność strony w wynikach wyszukiwania <br />
        <br />
        Optymalizacją kątem prędkości ładowania, uwzględniając wytyczne Core Web
        Vitals, co pozytywnie wpływa na doświadczenie użytkownika i ranking
        strony w wyszukiwarkach.
      </>
    }
    icon={<SEOServiceIcon />}
    data-aos="fade-right"
    data-aos-duration="1500"
  />
);
