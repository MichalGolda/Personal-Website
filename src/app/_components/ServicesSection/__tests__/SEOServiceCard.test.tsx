import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import SEOServiceCard from "../SEOServiceCard";

describe("SEOServiceCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<SEOServiceCard />);

    expect(container).toMatchSnapshot();
  });
});
