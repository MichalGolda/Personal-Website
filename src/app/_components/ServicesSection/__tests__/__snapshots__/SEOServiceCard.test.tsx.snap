// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEOServiceCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4"
    data-aos="fade-right"
    data-aos-duration="1500"
  >
    <div
      class="flex flex-row gap-x-4 md:flex-col md:gap-y-2 lg:flex-row md:items-start lg:items-center items-center"
    >
      <img
        alt="seo"
        class="w-[48px] lg:w-[64px]"
        data-nimg="1"
        decoding="async"
        height="64"
        loading="lazy"
        src="/_next/image?url=%2Fseo.webp&w=128&q=75"
        srcset="/_next/image?url=%2Fseo.webp&w=64&q=75 1x, /_next/image?url=%2Fseo.webp&w=128&q=75 2x"
        style="color: transparent;"
        width="64"
      />
      <h4
        class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
      >
        Optymalizacja treści (SEO) oraz szybkości ładowania
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      Badanie słów kluczowych i ich strategiczne umieszczanie w treściach, aby poprawić widoczność strony w wynikach wyszukiwania 
      <br />
      <br />
      Optymalizacją kątem prędkości ładowania, uwzględniając wytyczne Core Web Vitals, co pozytywnie wpływa na doświadczenie użytkownika i ranking strony w wyszukiwarkach.
    </p>
    <span
      class="block bg-primary w-32 h-0.5"
    />
  </div>
</div>
`;
